.env.example

```diff
@@ -32,11 +32,11 @@ REACT_APP_WEB3_TRANSPORT_PROVIDER=
 
 # Zoom (Face Recognition / Liveness Test API) options
 # Encrytption key
-REACT_APP_ZOOM_ENCRYPTION_KEY=
+REACT_APP_FACETEC_ENCRYPTION_KEY=
```

Comment:
lets keep the old env keys, this might cause lots of issues if we forget to update somewhere

Reply:
if we keep them let's leave a comment explaining that this is an env var that is not in use anymore and should be removed sometime in the future

Reply:
have reverted to the old env vars